components/Chips/src/MDCChipView.m

```diff
@@ -828,4 +876,20 @@ - (BOOL)willChangeSizeWithSelectedValue:(BOOL)selected {
   return !hasImage && hasSelectedImage;
 }
 
+- (void)rippleViewTouchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
+  [_rippleView touchesBegan:touches withEvent:event];
```

Comment:
Please use `self.rippleView` instead of accessing the iVar directly.  Since it's a property, the best practice is to use the getter whenever possible.

Reply:
done